types/pino/index.d.ts

```diff
@@ -561,8 +563,23 @@ declare namespace P {
         timestampKey?: string;
         /**
          * Format output of message, e.g. {level} - {pid} will output message: INFO - 1123 Default: `false`.
+         *
+         * @example
+         * ```js
+         * // You also can pass a function to have grained fine customization.
+         * const logger = pino({
+         *   prettyPrint: {
+         *     ignore: 'requestId',
+         *     messageFormat: (log, messageKey) => {
+         *       const message = log[messageKey];
+         *       if (log.requestId) return `[${log.requestId}] ${message}`;
+         *       return message;
+         *     },
+         *   },
+         * });
```

Comment:
This is example is far more detailed than it needs to be. I think we can simplify it, following [the example in the `pino-pretty` documentation](https://github.com/pinojs/pino-pretty#options).
```suggestion
         * {
         *   messageFormat: (log, messageKey) => {
         *     const message = log[messageKey];
         *     if (log.requestId) return `[${log.requestId}] ${message}`;
         *     return message;
         *   }
         * }
```